kubectl-cloudflow/cmd/deploy.go

```diff
@@ -97,14 +97,14 @@ func (opts *deployOptions) deployImpl(cmd *cobra.Command, args []string) {
 	version.FailOnProtocolVersionMismatch()
 
 	imageRef := args[0]
-	imageReference, err := parseImageReference(imageRef)
+	imageReference, err := verify.ParseImageReference(imageRef)
```

Comment:
functions like `ParseImageReference` are sort of reusable ones that can be used in multiple modules. We can assume that `deploy` can logically depend on `verify` - hence this makes sense. 👍

Reply:
would a `blueprint` package not make more sense than a `verify` package?

Reply:
@RayRoestenburg The pattern I followed here is cmd name -> package name as people have done before. Since we are planning to re-structure the folders/packages then that is an option to have s/verify/blueprint but not required if you ask me. I would have the blueprint and other high level stuff under a `cloudflow` package and then map commands to packages as usual, check kubectl for example here: https://github.com/kubernetes/kubectl/tree/master/pkg.

Reply:
yes it is fine for now.